terra-jupyter-base/Dockerfile

```diff
@@ -21,14 +21,15 @@ RUN sudo -i \
 # see article: https://developer.nvidia.com/blog/updating-the-cuda-linux-gpg-repository-key/
 # first we remove the bad keys from base image (https://github.com/NVIDIA/nvidia-docker/issues/1631#issuecomment-1112828208)
 RUN rm /etc/apt/sources.list.d/cuda.list \
-    && rm /etc/apt/sources.list.d/nvidia-ml.list \
```

Comment:
Is this removed because the list is not generated anymore?

Reply:
afaik yes, when the build failed with that line because it couldn't fine that .list